modules/must-gather-storage-limit.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="must-gather-storage-limit_{context}"]
+= Changing the must-gather storage limit
+
+When using the `oc adm must-gather` command to collect data the default maximum storage for the information is 30% of the storage capacity of the container. Once the 30% limit is reached the container is killed and the gathering process stops. Information already gathered will be downloaded to your local storage. You will need either a container with more storage capacity or to adjust the maximum volume percentage before running the must-gather command again.
+
+If you reach the storage limit you will see a message output informing you what happened.
+
+For example:
+
+[source,terminal]
+----
+Disk usage exceeds the volume percentage of %d for mounted directory. Exiting...
+----
+
+.Prerequisites
+
+ifndef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `cluster-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+ifdef::openshift-rosa,openshift-dedicated[]
+* You have access to the cluster as a user with the `dedicated-admin` role.
+endif::openshift-rosa,openshift-dedicated[]
+* The {product-title} CLI (`oc`) is installed.
+
+[IMPORTANT]
+====
+If you installed an earlier version of the `oc` binary, you cannot use it to run this command. You must download and install the 4.15 or later version of the `oc` binary.
```

Comment:
I'm waffling on whether this note is truly necessary or not. I know that people can sometimes use old oc's, but they can't assume that the old oc's will work with all new features. We don't necessarily call out in the docs each time we add something new to the CLI that they need to use the new CLI for. These docs are only being added to 4.15, so I think that should be sufficient. 

IMO I don't think it's necessary, but if you feel strongly you need to keep it, then that's fine.

Reply:
I took it out. I don't feel strongly enough to keep it in.